lib/offline/offline_utils.js

```diff
@@ -47,7 +47,9 @@ shaka.offline.OfflineUtils.getStoredContent = function(manifest) {
         language: stream.language,
         kind: stream.kind || null,
         width: stream.width,
-        height: stream.height
+        height: stream.height,
+        frameRate: stream.frameRate || null,
```

Comment:
You'll need to add frameRate and codecs to the externs for streams stored in the database (shakaExtern.StreamDB), and you'll need to add these properties to the code which stores streams offline (shaka.offline.Storage.prototype.createStream_).